tests/component/test_task_properties.py

```diff
@@ -35,3 +36,11 @@ def test___get_devices___shared_interpreter(ai_task: Task):
     devices = ai_task.devices
 
     assert all(dev._interpreter is ai_task._interpreter for dev in devices)
+
+
+def test___add_global_channels___returns_channels(task: Task):
```

Comment:
This file is for property get/set/reset tests. Functions like add_global_channels should be tested in test_task.py

Reply:
Moved.